MiniApp.xcodeproj/project.pbxproj

```diff
@@ -17,6 +17,8 @@
 		607FACDB1AFB9204008FA782 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 607FACD91AFB9204008FA782 /* Main.storyboard */; };
 		607FACE01AFB9204008FA782 /* LaunchScreen.xib in Resources */ = {isa = PBXBuildFile; fileRef = 607FACDE1AFB9204008FA782 /* LaunchScreen.xib */; };
 		607FACEC1AFB9204008FA782 /* Tests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 607FACEB1AFB9204008FA782 /* Tests.swift */; };
+		C3866B58983DE5A1C9FC45CF /* Pods_MiniApp_Tests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6E1CC4976FCCFA9C5737280C /* Pods_MiniApp_Tests.framework */; };
+		F8673656152B3624D9E6EAAB /* Pods_MiniApp_Example.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = FB348493F8CB435AB6C5F86C /* Pods_MiniApp_Example.framework */; };
```

Comment:
We don't usually commit with Pods integrated but it's up to you if this is easier. There are pros/cons to both approaches.

Reply:
Sure. Will consider this in the next commit.